src/Api/Data/StoryblokLink.php

```diff
@@ -15,5 +18,15 @@ public function __construct (array $data)
 		$this->id = $data["id"];
 		$this->uuid = $data["uuid"];
 		$this->slug = $data["slug"];
+		$this->isFolder = $data["is_folder"];
+		$this->position = $data["position"];
+		$this->name = $data["name"];
```

Comment:
Proposal: Rename `name` to `label`, or is that confusing?

Reply:
We have decided to leave it at `name`.